src/data/graphql/typeDefs.ts

```diff
@@ -18,6 +18,16 @@ export default gql`
     networkId: Int
   }
 
+  input NetworkContractsInput {
+    version: String
+    feeInverse: String
```

Comment:
are those for sure optional?

Reply:
Yes.

Since this is coming from the contracts, the might have issues connecting, through the provider, to the network, in which case these values will be undefined.